jenkins/cut-release

```diff
@@ -1,13 +1,18 @@
 #!/bin/bash
-# Parameters:
+
+_required_variables='WHICH_REPO REPO_BASE'
+for _variable in ${_required_variables}; do
+    test -n "${!_variable}" || { echo "$_variable must be set"; exit 1; }
+done
+
+# optional variables
 : ${REMOTE:=origin}
 : ${BRANCH_NEW:=master}
 : ${BRANCH_OLD:=rc}
-: ${SHOULD_CUT_BRANCH:=0}
-: ${BACKPORT_TO_MASTER:=0}
+: ${SHOULD_CUT_BRANCH:=$(false)}
 : ${BRANCH_TO_CUT:=rc}
 : ${WORKSPACE:=${HOME}/workspace/cut_branch}
-REPO_BASE='Stanford-Online'
```

Comment:
This is no longer hard-coded, to ease development.
Instead, it's passed in as a variable.